build.sbt

```diff
@@ -56,20 +57,23 @@ lazy val common = project.in(file("faunadb-common"))
     javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
     apiURL := Some(url(s"http://faunadb.github.io/faunadb-jvm/$driverVersion/faunadb-common/api/")),
 
-    (javacOptions in doc) := Seq("-source", "1.7", "-Xdoclint:none",
```

Comment:
Did you reenable the doc lint? Are the doc compiling? I remember we've disabled due to the amount of useless warnings that we decided we didn't want to fix.